app/controllers/BankDetailsController.scala

```diff
@@ -286,7 +288,10 @@ class BankDetailsController @Inject() (
 
   def barsLockout(p: String): Action[AnyContent] = as.default { implicit request =>
     val charset = Charset.forName("UTF-8")
-    val expiry = new String(java.util.Base64.getDecoder.decode(p.getBytes(charset)), charset)
+    val expiry = Try[String] {
+      new String(java.util.Base64.getDecoder.decode(p.getBytes(charset)), charset)
+    }.getOrElse(Instant.now.longFormat)
```

Comment:
I think it's preferable to error out if `p` cannot be decoded - we always expect it to be so and providing the time now could be confusing and hide the error from us in case something did go wrong and it's our fault and not the user playing around with the parameter